Resources/Announcements/modNews-zh_CN.json

```diff
@@ -77,7 +77,7 @@
         "\n     - 日语 (By: Sunnyboi)",
         "\n     - 拉丁美洲语 (By: CreepPower)",
         "\n     - 俄语 (By: TommyXL, Shoulder Devil, chill_ultimated, Nevermore59)",
-        "\n     - 简体中文 (By: 船员小青，乐崽吖，绿色游戏(NikoCat233))",
+        "\n     - 简体中文 (By: 青翰，乐崽吖，绿色游戏(NikoCat233))",
         "\n     - 西班牙语 (By: Dawson, Sunnyboi, thewhiskas27, xxSShadow)",
```

Comment:
Chenge this in Crowdin

Reply:
The only lines where you can make changes through PR are only in `en_US.json` or `modNews-en_US.json`